addons/storyshots/storyshots-core/src/Stories2SnapsConverter.js

```diff
@@ -24,9 +24,18 @@ class DefaultStories2SnapsConverter {
   }
 
   getSnapshotFileName(context) {
-    const { fileName } = context;
+    const { fileName, kind } = context;
 
     if (!fileName) {
+      // eslint-disable-next-line no-console
+      console.warn(
+        `Storybook was unable to detect filename for stories of kind "${kind}".
```

Comment:
should use `ts-dedent` here?

Reply:
But this is a `.js` not a `.ts` file. Is it OK?